authors/apps/articles/serializers.py

```diff
@@ -8,14 +9,29 @@
 
 class ArticleSerializer(serializers.HyperlinkedModelSerializer):
     id = serializers.IntegerField(read_only=True)
-    title = serializers.CharField(required=False, allow_blank=True, max_length=100)
+    title = serializers.CharField(
+        required=False,
+        allow_blank=True,
+        max_length=100)
```

Comment:
try making the title mandatory because the title is intended to provide a summarised description of the purpose or scope of the article.

Reply:
Done.